src/cgame/cg_buildable.cpp

```diff
@@ -1366,11 +1366,23 @@ static void CG_BuildableParticleEffects( centity_t *cent )
 CG_BuildableStatusParse
 ==================
 */
+static bool PC_String_Parse( int handle, qhandle_t& qhandle )
+{
+	pc_token_t token;
+
+	if ( !Parse_ReadTokenHandle( handle, &token ) )
+	{
+		return false;
+	}
+
+	qhandle = trap_R_RegisterShader( token.string, static_cast<RegisterShaderFlags_t>( RSF_NOMIP ) );
```

Comment:
Why is there something about registering shaders in a function called "string parse"?